plan/common_plans.go

```diff
@@ -407,51 +406,97 @@ type Explain struct {
 	explainedPlans map[int]bool
 }
 
-// prepareExplainInfo4DAGTask generates the following information for every plan:
-// ["id", "parents", "task", "operator info"].
-func (e *Explain) prepareExplainInfo4DAGTask(p PhysicalPlan, taskType string, parentID string) {
-	childrenIDs := make([]string, 0, len(p.Children()))
-	for _, ch := range p.Children() {
-		childrenIDs = append(childrenIDs, ch.ExplainID())
+const (
+	// treeBody indicates the operator tree is not finished.
+	treeBody rune = '│'
+	// treeMiddleNode indicates this operator is in the middle the operator tree.
```

Comment:
`in the middle the operator tree` is puzzled. `not the last child of its parent` is clearer.